bin/console

```diff
@@ -7,7 +7,7 @@ require "ingram_micro"
 # with your gem easier. You can also use a different console, if you like.
 
 # (If you use this, don't forget to add pry to your Gemfile!)
-# require "pry"
+require "pry"
```

Comment:
The gem isn't installed in production and so will probably break `rails console` on prod